src/etc/platform-intrinsics/powerpc.json

```diff
@@ -72,6 +72,55 @@
             "llvm": "vmin{0.kind}{0.data_type_short}",
             "ret": "i(8-32)",
             "args": ["0", "0"]
+        },
+        {
+            "intrinsic": "sub{0.kind}{0.data_type_short}s",
+            "width": [128],
+            "llvm": "vsub{0.kind}{0.data_type_short}s",
+            "ret": "i(8-32)",
+            "args": ["0", "0"]
+        },
+        {
+            "intrinsic": "subc",
+            "width": [128],
+            "llvm": "vsubcuw",
+            "ret": "u32",
+            "args": ["0", "0"]
+        },
```

Comment:
VEC_SUBC [here](http://openpowerfoundation.org/wp-content/uploads/resources/leabi/content/dbdoclet.50655245_56497.html) alludes to both unsigned and signed version existing. Is it intended to only expose unsigned version with the canonical name `subc`?